scripts/ci-entrypoint.sh

```diff
@@ -250,8 +248,16 @@ export ARTIFACTS="${ARTIFACTS:-${PWD}/_artifacts}"
 # create cluster
 create_cluster
 
-# install CNI and CCM
-install_addons
+# export the target cluster KUBECONFIG if not already set
+export KUBECONFIG="${KUBECONFIG:-${PWD}/kubeconfig}"
```

Comment:
is this net new? if so how is this working for the similarly implemented cloud-provider-azure tests that rely upon ci-entrypoint.sh?

Reply:
This is factored out of `install_addons` so it runs even in the cases where `install_addons` no longer will with these changes.